index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <h2><a href="#">Welcome</a></h2>
+        <i class="fa fa-bars" aria-hidden="true"></i>
+    </header>
+    <section class="home-page">
```

Comment:
- [ ] Like I said before your design really well matches the Figma 👍. However, as part of the requirements, we will need you to use `flexbox` to align this `section` tag. this is to ensure you have a good understanding of how to layout the content using flexbox